### ISSUE TITLE
Unexpected token 'U', "Unexpected"... is not valid JSON

### ISSUE THREAD

**Issue by ddefia** [stackblitz/bolt.new#1474]
### Describe the bug

Unexpected token 'U', "Unexpected"... is not valid JSON

I can not load any project in.

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-xvtbj7

### Steps to reproduce

Try and load project

### Expected behavior

The project should load in

### Platform

```
Browser name = Chrome
Full version = 130.0.0.0
Major version = 130
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 470784550,
  "usedJSHeapSize": 434870918,
  "jsHeapSizeLimit": 4294705152
}
Username = ttony15
Chat ID = a00995aaad76
```

### Additional context

Error:
```
An unknown error occured while running an action.
```

**Comment by YouZag** [comment]
Same.  Subscribed and now I can't do anything.  Unexpected token 'U', "Unexpected"... is not valid JSON

**Comment by buligAi** [comment]
Same. I cry! I wasted 1M tokens already for that app.

**Comment by kc0tlh** [comment]
@YouZag @ddefia @buligAi Thanks for the report and apologies for the frustrating experience! The fix is explained in #1471
